client/src/components/molecules/RequestsTable.tsx

```diff
@@ -38,6 +38,7 @@ const RequestsTable: FunctionComponent<Props> = (props: Props) => {
     const [requests, setRequests] = useState(props.requests.filter((request) => request.deleted === false));
 
     useEffect(() => {
+        console.log("hoi");
```

Comment:
stray console.log

ctrl+f for console.logs your diffs on github before request review to avoid comments like this